docker/airflow/Dockerfile

```diff
@@ -4,10 +4,9 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
 RUN curl https://packages.microsoft.com/config/debian/11/prod.list > /etc/apt/sources.list.d/mssql-release.list
 RUN apt-get update \
     && apt-get install -y \
-        gcc libsasl2-dev gnupg build-essential libssl-dev libffi-dev \
-        librdkafka-dev unixodbc-dev libxml2 libevent-dev --no-install-recommends  \
+        build-essential gcc gnupg libevent-dev libffi-dev libpq-dev librdkafka-dev libsasl2-dev libsasl2-modules  \
```

Comment:
looks like we are missing on `openjdk-11-jre` that is required for deltalake connector! Let's add that in here!

Reply:
Should I add also the same dependencies that we have in `ingestion/Dockerfile`?

Reply:
we can ignore `python3.9-dev`, `netcat` and `wget`